{"name": "test_get_universe_invalid_vendor", "status": "passed", "description": "\n        Verify that incorrect vendor name is recognized.\n        ", "attachments": [{"name": "log", "source": "d6cf4b39-362a-4b91-b499-dc59f8964d82-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "7a04f67a-3925-4620-acee-f15dfa469215-attachment.txt", "type": "text/plain"}], "start": 1656452256528, "stop": 1656452256529, "uuid": "2354f648-c22c-4b94-96d9-0f2bc4243dcd", "historyId": "dc1345c7a0630fb04a78204864dfa99f", "testCaseId": "5c8cdd8dd1499c57e01d702e6641259f", "fullName": "im_v2.common.universe.test.test_universe.TestGetUniverseGeneral1#test_get_universe_invalid_vendor", "labels": [{"name": "parentSuite", "value": "im_v2.common.universe.test"}, {"name": "suite", "value": "test_universe"}, {"name": "subSuite", "value": "TestGetUniverseGeneral1"}, {"name": "host", "value": "8331b618a6a7"}, {"name": "thread", "value": "10537-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "im_v2.common.universe.test.test_universe"}]}